src/mappings/uniques/mint.ts

```diff
@@ -68,9 +68,12 @@ export async function handleTokenCreate(context: Context): Promise<void> {
   success(OPERATION, `${final.id}`)
   await context.store.save(final)
   await context.store.save(collection)
-  await createEvent(final, OPERATION, event, '', context.store)
-
-  if (final.issuer !== final.currentOwner) {
-    await createEvent(final, Action.SEND, event, final.currentOwner, context.store, final.issuer)
-  }
+  await createEvent(
+    final,
+    OPERATION,
+    event,
+    final.issuer !== final.currentOwner ? final.currentOwner : '',
```

Comment:
`const caller = ` so you do not need to write ternary twice. 

DRY

Reply:
could you rephrase or add suggestion?

Reply:
updated in [717202a](https://github.com/kodadot/stick/pull/192/commits/717202aa94a826a43eb1e104f7e7b6f3e65455e4)